### PR TITLE
Fix getLPTokenExchangeValue, no need to use Maths library as precisio…

### PR DESCRIPTION
**Pull request by grandizzy**
…n match up collateral and quote tokens

### CODE REVIEW

**Review by MikeHathaway — APPROVED** [review]
LGTM

File: src/ERC20Pool.sol [hunk 1]
```diff
@@ -624,10 +624,10 @@ contract ERC20Pool is IPool, Clone {
         ) = bucketAt(_price);
 
         // calculate lpTokens share of all outstanding lpTokens for the bucket
-        uint256 lenderShare = PRBMathUD60x18.div(_lpTokens, lpOutstanding);
+        uint256 lenderShare = _lpTokens / lpOutstanding;
```
Comment (EdNoepel) [inline src/ERC20Pool.sol:627]: ray/ray -> unscaled result?
Wouldn't this truncate decimal places?

File: src/ERC20Pool.sol [hunk 2]
```diff
@@ -624,10 +624,10 @@ contract ERC20Pool is IPool, Clone {
...
 
-        collateralTokens = PRBMathUD60x18.mul(bucketCollateral, lenderShare);
-        quoteTokens = PRBMathUD60x18.mul(onDeposit + debt, lenderShare);
+        collateralTokens = bucketCollateral * lenderShare;
```
Comment (EdNoepel) [inline src/ERC20Pool.sol:629]: ray*unscaled -> ray
Since we're using solc > 0.8, we have overflow protection built-in.

File: src/ERC20Pool.sol [hunk 3]
```diff
@@ -624,10 +624,10 @@ contract ERC20Pool is IPool, Clone {
...
+        quoteTokens = (onDeposit + debt) * lenderShare;
```
Comment (EdNoepel) [inline src/ERC20Pool.sol:630]: (rad+rad)*unscaled -> rad

**Review by EdNoepel — COMMENTED** [review]
I don't understand how this works.  It seems to divide a `ray` by a `ray`, we'd first need to scale the LHS by 10**27 to preserve precision.  Would be a lot easier to understand if the numeric types were commented.

File: src/ERC20Pool.sol (hunk 1 above)
Comment (grandizzy) [reply src/ERC20Pool.sol:627]: yeah, nm, please disregard this one

**Comment by MikeHathaway** [comment]
Updated this in https://github.com/ajna-finance/contracts/pull/75 as well

**Pull request closed without merging**